src/core/sys/linux/netinet/in_.d

```diff
@@ -107,7 +115,7 @@ version(CRuntime_Glibc)
     enum IN6ADDR_ANY_INIT      = in6_addr.init;
     enum IN6ADDR_LOOPBACK_INIT = in6_addr([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]);
 
-    static if (__USE_MISC)
+    version(gnu_libc) static if (__USE_MISC)
```

Comment:
Not sure why you chose this version name, plus don't think this will work for uClibc, as `__USE_MISC` is undefined there.  You may have seen it in `core.sys.linux.config` but this module invokes `core.sys.posix.config`, which doesn't have it.

Reply:
uClibc generally follows Glibc, figured that they both got the GNU roots so that's the best name I could come up with...

'__USE_MISC' is defined in `core.sys.posix.config` for both Glibc and uClibc.

Reply:
OK, name is fine, but [I don't see that enum defined for uClibc](https://github.com/dlang/druntime/blob/master/src/core/sys/posix/config.d#L80).

Reply:
OK, they were on my setup... fixed in https://github.com/dlang/druntime/pull/2143